crates/macro-support/src/parser.rs

```diff
@@ -21,9 +21,12 @@ thread_local!(static ATTRS: AttributeParseState = Default::default());
 
 /// Javascript keywords.
 ///
+/// Note that some of these keywords are only reserved in strict mode. Since we
+/// generate strict mode JS code, we treat all of these as reserved.
+///
 /// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#reserved_words
-const JS_KEYWORDS: [&str; 44] = [
-    "arguments", // reserved in strict mode, which we are in
+const JS_KEYWORDS: [&str; 47] = [
```

Comment:
Hm, seeing that this is a superset of `VALUE_LIKE_JS_KEYWORDS` I would have preferred it to include it programmatically. But I'm afraid Rust doesn't give us the tools for that.

WDYT of removing the corresponding keywords from that list and include them programmatically correctly in `is_js_keyword()`?

Reply:
Not a fan. The idea of a list that contains all keywords, except all keywords in this other list, seems weird to me. I also don't think the code duplication is bad enough to justify it.

Reply:
Sure, leave a comment at `VALUE_LIKE_JS_KEYWORDS` then saying that `JS_KEYWORDS` is a superset.